lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -219,10 +213,15 @@ private JourneyResponse getJourneyResponse(
             final boolean isF2FIncomplete = !Objects.isNull(f2fRequest) && !hasF2fVc;
             final boolean isF2FComplete = !Objects.isNull(f2fRequest) && hasF2fVc;
 
-            var ciScoringCheckResponse =
+            Optional<JourneyResponse> ciScoringCheckResponse =
```

Comment:
Interesting, I prefer using `var` for local variables - the IDE and compiler still know the type, but it's less verbose and makes refactoring easier!

I don't think it's a big deal though (some day I might write a style guide, but today is not that day)